fastlane/Fastfile

```diff
@@ -16,6 +16,11 @@ platform :ios do
       UI.message "Running git status check"
       ensure_git_status_clean(show_uncommitted_changes: true, show_diff: true)
     end
+    if lane != :test
```

Comment:
Don't we want to run it before git status check?
To make sure bootstrap does not make our repo dirty

Reply:
Good idea!